includes/specials/SpecialUserlogin.php

```diff
@@ -665,32 +650,6 @@ public function authenticateUserData() {
 			return self::SUCCESS;
 		}
 
-		$this->mExtUser = ExternalUser_Wikia::newFromName( $this->mUsername );
-
-		global $wgEnableHeliosExt;
-		if ( $wgEnableHeliosExt ) {
-			\Wikia\Helios\User::debugLogin( $this->mPassword, __METHOD__ );
-		}
-
-		global $wgExternalAuthType;
-		if ( $wgExternalAuthType
-		&& is_object( $this->mExtUser )
-		&& $this->mExtUser->authenticate( $this->mPassword ) ) {
-			# The external user and local user have the same name and
-			# password, so we assume they're the same.
-			$this->mExtUser->linkToLocal( $this->mExtUser->getId() );
-		}
-
-		// Wikia change - begin - author: @wladek
-		if ( $wgExternalAuthType
-			&& is_object( $this->mExtUser )
-			&& $this->mExtUser->getLastAuthenticationError() )
-		{
-			$this->mAbortLoginErrorMsg = $this->mExtUser->getLastAuthenticationError();
-			return self::ABORTED;
-		}
-		// Wikia change - end
-
 		# TODO: Allow some magic here for invalid external names, e.g., let the
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)